pkg/volume/persistent_claim/persistent_claim.go

```diff
@@ -80,6 +81,19 @@ func (plugin *persistentClaimPlugin) NewBuilder(spec *volume.Spec, pod *api.Pod,
 		return nil, err
 	}
 
+	// If a GID annotation is provided set the GID attribute.
+	if volumeGid, ok := pv.Annotations["GID"]; ok {
```

Comment:
+1 for documentation and examples. And IMO we should not use magic annotations, it should be covered by real PV attribute eventually. I.e. start with "alpha" annotation and convert to attribute when proven to be useful.